core/db.py

```
@@ -106,7 +106,7 @@ def execute(self, statement: Any, *args: Any, **kwargs: Any):
             result = conn.execute(stmt, *args, **kwargs)
             try:
                 conn.commit()
-            except Exception:
+            except Exception:  # noqa: BLE001
                 # Not all statements require/allow commit; ignore commit errors
                 pass
```

Ruff (0.13.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)